diesel/src/pg/types/floats/mod.rs

```diff
@@ -43,12 +43,12 @@ impl Error for InvalidNumericSign {
 impl FromSql<types::Numeric, Pg> for PgNumeric {
     fn from_sql(bytes: Option<&[u8]>) -> Result<Self, Box<Error+Send+Sync>> {
         let mut bytes = not_none!(bytes);
-        let ndigits = try!(bytes.read_u16::<BigEndian>());
-        let mut digits = Vec::with_capacity(ndigits as usize);
+        let num_digits = try!(bytes.read_u16::<BigEndian>());
```

Comment:
This is objectively a better name. It's worth noting that I named these local variables for parity with PG internally though, which is what this is based off of. https://github.com/postgres/postgres/blob/6928484bda454f9ab2456d385b2d317f18b6bf1a/src/backend/utils/adt/numeric.c#L822-L825

I don't feel strongly about whether that matters or what this should be named, but I just wanted to mention it. We should go with whichever you think is best.